### PR TITLE
[#503] Add umd build

### LINKED ISSUE #503: UMD
Hey Guys,

would you be so kind to provide us with an UMD-version? I want to introduce these great pickers to my team and this would really help me alot :) Is this possible? I´d really appreciate it. 

Greetings
Dave

### PR DESCRIPTION
**Pull request by dmtrKovalenko**
- [x] I have changed my target branch to **develop** :facepunch: 

## Description
Adds umd build

### CODE REVIEW

File: lib/rollup.config.js [hunk 1]
```diff
@@ -1,36 +1,82 @@
 import babel from 'rollup-plugin-babel';
 import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
+import { uglify } from 'rollup-plugin-uglify';
 import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
 import pkg from './package.json';
 
 // treat as externals not relative, not absolute and not reserved rollup paths
-const external = id =>
+const cjsExternal = id =>
```
Comment (TrySound) [inline lib/rollup.config.js]: It's not only `cjs`.

File: lib/rollup.config.js [hunk 2]
```diff
@@ -1,36 +1,82 @@
...
+    external: Object.keys(globals),
+    output: {
+      globals,
+      format: 'umd',
+      name: pkg.name,
+      sourcemap: true,
+      file: 'build/dist/material-ui-pickers.umd.js',
+    },
+    plugins: [
+      resolve({ extensions: ['.js', '.jsx'] }),
```
Comment (TrySound) [inline lib/rollup.config.js]: I just reused this stuff

File: lib/rollup.config.js [hunk 3]
```diff
@@ -1,36 +1,82 @@
...
-    }),
-    sizeSnapshot()
+const input = './src/index.js';
+const globals = {
+  react: 'React',
+  'react-dom': 'ReactDOM',
+  'prop-types': 'PropTypes',
+};
+
+const babelOptions = {
```
Comment (TrySound) [inline lib/rollup.config.js]: This was a function for a future reason.

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 2
```
Comment (TrySound) [inline .editorconfig]: prettier is bad?

File: lib/rollup.config.js (hunk 2 above)
Comment (dmtrKovalenko) [reply lib/rollup.config.js]: :D

File: .editorconfig (hunk 4 above)
Comment (dmtrKovalenko) [reply .editorconfig]: 💃

File: lib/rollup.config.js (hunk 3 above)
Comment (dmtrKovalenko) [reply lib/rollup.config.js]: 👍

**Pull request merged**